patches/0070-Temporarily-undo-aws.iam.Role-managed_policy_arns-an.patch

```diff
@@ -0,0 +1,32 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Anton Tayanovskyy <anton@pulumi.com>
+Date: Fri, 25 Oct 2024 13:17:15 -0400
+Subject: [PATCH] Temporarily undo aws.iam.Role managed_policy_arns and
```

Comment:
Do we already have a ticket for tracking how we wanna tackle this long term?

Reply:
Yes. https://github.com/pulumi/pulumi-aws/issues/4654 tracks figuring this out as soon as time permits.